### PR TITLE
Fix E2Es failing on master due to common not found element(s)

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
```release-note
NONE
```

### CODE REVIEW

**Review by nevyangelova — APPROVED** [review]

**Comment by nevyangelova** [comment]
@yasserfaraazkhan you got a timeout, running again

**Comment by yasserfaraazkhan** [comment]
> @yasserfaraazkhan you got a timeout, running again

@nevyangelova Yeah, 
saw this for first time...will try to debug

File: e2e/cypress/tests/support/ui/global_header.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ Cypress.Commands.add('uiOpenProductMenu', (item = '') => {
 });
 
 Cypress.Commands.add('uiGetSetStatusButton', () => {
-    return cy.findByRole('button', {name: 'set status'}).should('be.visible');
+    return cy.get('.status-wrapper > .Avatar').should('be.visible');
```
Comment (M-ZubairAhmed) [inline e2e/cypress/tests/support/ui/global_header.js:30]: why did we had to do this way? did the semantics change if yes should we not try to use testing principles :P

**Pull request closed without merging**